09-echo_data.ipynb

```diff
@@ -18,10 +18,11 @@
     "from res.sequential_tasks import EchoData\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import numpy as np
</pre><p>You've already imported <code>numpy</code>…</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Atcold/pytorch-Deep-Learning/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Deleted